packages/@aws-cdk-testing/cli-integ/resources/templates/autoscaling-template.yml

```diff
@@ -0,0 +1,677 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
Where is this being used?

Reply:
It's not as it doesn't pass for csharp. Had planned to have multiple integ test files for the various languages but these aren't being used until csharp passes generally

Reply:
I wanted to use them as tests for our languages so we had a more robust set of tests as the current one does not test much. But given not all languages can handle all templates just yet. I will remove for now and add again later